modules/my_health/spec/request/v1/medical_records/radiology_request_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/mr_client_helpers'
+require 'medical_records/client'
+require 'medical_records/bb_internal/client'
+
+RSpec.describe 'Medical Records Session', type: :request do
+  include MedicalRecords::ClientHelpers
+  include SchemaMatchers
+
+  let(:va_patient) { true }
+  let(:mhv_account_type) { 'Premium' }
+  let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
+
+  before do
+    allow(MedicalRecords::Client).to receive(:new).and_return(authenticated_client)
+    allow(BBInternal::Client).to receive(:new).and_return(BBInternal::Client.new(
+                                                            session: { user_id: 15_176_497,
+                                                                       patient_id: '15176498',
+                                                                       expires_at: Time.current + (60 * 60),
+                                                                       token: 'SESSION_TOKEN' }
+                                                          ))
+    sign_in_as(current_user)
+  end
```

Comment:
@mmoyer-va could you refactor line 18 better spacing/readability. For example:

```suggestion
before do
  bb_internal_client = BBInternal::Client.new(
    session: {
      user_id: 15_176_497,
      patient_id: '15176498',
      expires_at: Time.current + 1.hour,
      token: 'SESSION_TOKEN'
    }
  )

  allow(MedicalRecords::Client).to receive(:new).and_return(authenticated_client)
  allow(BBInternal::Client).to receive(:new).and_return(bb_internal_client)
  sign_in_as(current_user)
end
```

Reply:
@stevenjcumming thank you for the suggestion -- this is much better.